controllers/platformnetwork_controller.go

```diff
@@ -829,35 +824,38 @@ func (r *PlatformNetworkReconciler) IsPlatformNetworkInSync(client *gophercloud.
 		return false, err
 	}
 
-	return pool_update_required || nwk_update_required, err
+	return !(pool_update_required || nwk_update_required), err
```

Comment:
what's the change now?

Reply:
Previously the method was called `UpdatePlatformNetwork` and Yuxing suggested to change the name and it was renamed `IsPlatformNetworkInSync`, updated return value to match the name and method description.